SOLR-6572: Trim whitespace in XML nodes in solrconfig.xml (#2274)
apache/solr: solr/CHANGES.txt

* SOLR-17148: Fixing Config API overlay property enabling or disabling the cache (Sanjay Dutt, hossman, Eric Pugh)

> * SOLR-6572: Extra leading or trailing whitespace in XML nodes in solrconfig.xml not supported

Reviewer: ```suggestion
* SOLR-6572: Extra leading or trailing whitespace in XML nodes in solrconfig.xml is now ignored
```

or

```suggestion
* SOLR-6572: Extra leading or trailing whitespace in XML nodes in solrconfig.xml is now supported by being ignored
```
